### ISSUE TITLE
Superconductor changes

### ISSUE THREAD

**Issue by 0lafe** [GTNewHorizons/NewHorizons#4333]
#### Which modpack version are you using?
2.0.7

#### What do you suggest instead/what changes do you propose?

(SC means superconductor)
There's a bunch of "issues" with SC at the moment. MV-EV versions use T+1 pumps, while IV+ use tier pumps. The He/N cell count seems to go up or down each tier, without a consistent increase/decrease. The materials themselves seem to actually get easier some times, with the IV base being more expensive than LuV, and zpm. Even UV kinda, except UV gives the mk3 fusion which makes In easier, and therefor the IV superconductor is a little easier than UV as you have no access to pure Nq* ore in UV. The pump usage on them is also, sort of weird. It's kind of the primary cost, but it's a bit of a weird cost for a wire. 

First let's look at the amount made. MV does 3x at a time, HV too, EV does 9x, IV does 6x, LuV does 8x, zpm does 16x, UV does 36x (or 63x?) and UHV does 24x. It seems like a few were made based on +3/tier, but only MV/EV/UHV fit that. The others don't really make sense in their usage amount. I propose changing them to +3/tier so 3x for MV, 6x for HV, 9x for EV, etc. 

Next the gas cost. It's used in cells, which aren't returned. Which gives SC a weird PTFE/iron cost. If we stick to one type of gas per tier, it could be used in fluid form instead, removing the cell cost completely. 2B N2 for MV, 4B for HV, 8B for EV, 16B for IV, 2B He for LuV, 4B He for zpm, 8B He for UV and 16B He for UHV. Gives it a more consistent gas usage, as well as removes  the weird PTFE/iron cost. Another option for the gasses is to use amounts related to the temperatures the SC need. So colder temps = more gasses. Not sure if that's how that works at all, but it's sort of interesting. 

For the dust mixtures themselves, MV-EV seem fairly ok, but IV+ can do with a little change. IV being 3/4 In is really the worst cost of these for a while. Becuase of this, the LuV and ZPM SC become cheaper versions of the IV one, leaving the IV SC as the most expensive for a while. I looked at some other alternatives we could use that are more interesting and give more costly components for later tiers/a better IV one. For IV, the superconductor is currently VIn3, but the real SC is V3In. Changing it to that seems like a better option for IV. An alternate recipe is (Pu,In)CoGa5. It's a little more interesting of a compound to make, but is a little less costly. LuV could be AuIn3 which expands on the indium cost instead of decreasing it. ZPM could use (Pu-241,Ce)2(Ba,Nq+)2Cu3O10. Naquadria coming from centrifuging naquadah/enriched naq, and cerium coming from uu/neodynium veins/GT++ dim ores. It's a real SC recipe with Eu being replaced with Nq+ and Pu-241 because Eu is already used quite heavily, but Nq+/Pu-241 much less. The recipes kinda go into fantasy with ZPM+ anyway, so using less realistic recipes isn't the worst thing. 

Lastly, pumps. Using T+1 pumps for lower tiers almost works, but isn't really a sustainable progression unless we want to lock later tier superconductors past their tier, which would need some recipe changes, as well as usage of cheaper SC within a given tier. Using pumps of the tier is a little better, but ends up with the cost of the wire being primarily the cost of the pump. My idea is to tier the pumps down, but increase the amount to 2. So an IV SC will cost 2 EV pumps for every 12x wires produced. This is sort of a nerf to their cost, but is also aimed to keep their cost more in line with being a wire, as opposed to just a pump dump (lol). As lategame parts are used in smallish quantities outside of superconductors, it gives superconductors a weirdly high cost as they're mostly just an excessive use of higher tier components. 

EDIT: new thing. SC coils for fusion cost a little too much time because of 1. the non scaling of multiples for crafting which I addressed above, and 2. the SC count in coils is a little too high. 128x LuV per coil is way too much. Changing this to 64x LuV and 2x less for each tier after should solve this. Combined with the pump cost change in SC's, this brings the cost of the pumps just a little higher than the cost of the SC ingots. The total crafting time for an mk1 reactor with 1.5A ZPM is 17h. The crafting time with ~2A LuV is 68h, encouraging doable but high tier power setups. 

TL;DR
New recipes for dusts, 2x T-1 pumps per craft, 3xwires/tier done at a time, gases switched to fluids so no cells.

**Comment by DietmarKracht** [comment]
Lots of invalid pattern incoming

**Comment by botn365** [comment]
a single 6 quad cell reactor produses 0.48 plutonium/h wich wil be 3.36 ingots iv/super conductor witch is too expansif
(Sn5In)Ba4Ca2Cu10Oy is abetter it forces you to get some indium but not nearly as much i do like the idee of galium in ist so maby it can be  (Ga5In)Ba4Ca2Cu10Oy  or (Sn5In)Ga4Ca2Cu10Oy

for the new luv recipy you will now neet 1536 indium to build the first fusion reactor compared to the previus ~160 i tink for luv it shoud be 1In = 8 SC ingots so fusion is 192 Indium

for zpm you neet a tier 6 rocket wich is uv for good source of pu241 so when you are in zpm age you stil have to use the reactor wich at 0.48pu/h ->(1 pu = 0.2 pu241) 0.18133... zpm SC per/ hour 
maybe it shoud be normel pu here

**Comment by 0lafe** [comment]
>for zpm you neet a tier 6 rocket wich is uv for good source of pu241 so when you are in zpm age you stil have to use the reactor wich at 0.48pu/h ->(1 pu = 0.2 tiny pu241) 0.18133... zpm SC per/ hour
maybe it shoud be normel pu here

The IV rocket gives you pure Pu239 ore which can be centrifuged to 241. 

>for the new luv recipy you will now neet 1536 indium to build the first fusion reactor compared to the previus ~160 i tink for luv it shoud be 1In = 8 SC ingots so fusion is 192 Indium

Not saying no, but with the drill changes, a single vein of sphalerite/galena should give you more than that much indium from the ore recipe. You can supliment some with uu, for more than enough.

**Comment by botn365** [comment]
> The IV rocket gives you pure Pu239 ore which can be centrifuged to 241.

the Pu ore vein that can be found on T3 planets is the second rarest ore vein in GTNH it shoud not be a reqierment to progress in the game

for the indium you alsoo neet an insane amount of aluminium it go's from 9 828 aluminium to 
55 296 aluminium to build the first fusion reactor the recipy for luv SC at least 1indium -> 4 SC ingots so you dont neet insane amounts of materials

**Comment by botn365** [comment]
recipy for ZPM
(Eu,Ce)2(Nq+,Eu)2In3O10+

**Comment by botn365** [comment]
recipy for iv
(Pu,In)CoGa5

**Comment by 0lafe** [comment]
Put that for IV, but the zpm one idk. Eu is already used a lot in zpm itself, being in like everything through motors. UV also uses Eu as well. What about uh, Yb? or Y?

**Comment by botn365** [comment]
im tinking about La it doesn't have any uses now and change the rare eart recipy from 20% for small to 20% for tiny

**Comment by 0lafe** [comment]
La would be kinda fun, but all the La compounds seem to be super cheap NH wise. (La,Ba)2CuO4 seems about as good as they get. I guess it could be for something earlier, like HV? 

Yb is a GT++ thing. Only an Everglades ore rn, but alk said he wanted to give it another creation method. Not sure what it'll be, but I think it's intended to be pre Everglades, so IV or earlier. 

We could just make fake recipes of lots of rare Earth stuff put together, but idk. The realistic stuff substituted with some of our fake things seem cool to me. It would sorta be nice to give things like Nq/De/Sp stuff properties. So like Nq+ being a copper replacement, or Vx for gold or something. Just keeps the lore side of it intact lol

**Comment by botn365** [comment]
for La i was tinking for zpm as  (La,Ce)2(Nq+,La)2In3O10+ but it stils feel to cheap maby as (La,Ce)2(Nq+,La)2In3Ar10+ 
according to wikipedia La is extracted from monazite and bastnatide wich looks more complex then indium 

also for LuV SC i was tinking of infAu3In so you have to build a T4 rocket

**Comment by Dream-Master** [comment]
We will change SC for 8.0.

**Comment by Ngar1** [comment]
Direct supply gas of a spezific among to an assembler by ME ist on my nowledge not possible. May be over cells canning/uncanning - but than is the old receipt better.

**Comment by boubou19** [comment]
you can if you encode your pattern with a fluid container that hold exactly the amount you need then use the encoded pattern in a fluid interface

**Comment by 0lafe** [comment]
Difference between this and old cell recipe is using the fluid doesn't consume the cell. There's other ways to do it to input fluid amounts with recipe as well, quite doable.
